<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMSAT</class>
 <widget class="QDialog" name="DialogMSAT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>309</width>
     <height>320</height>
    </rect>
   </property>
   <property name="title">
    <string>MSAT</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>alphaï¼š</string>
          </property>
          <property name="buddy">
           <cstring>BoxMSATalpha</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="BoxMSATalpha">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.000100000000000</double>
          </property>
          <property name="value">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>beta:</string>
          </property>
          <property name="buddy">
           <cstring>BoxMSATalpha</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="BoxMSATbeta">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.000100000000000</double>
          </property>
          <property name="value">
           <double>0.005000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>epsilon:</string>
          </property>
          <property name="buddy">
           <cstring>BoxMSATalpha</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="BoxMSATepsilon">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.000100000000000</double>
          </property>
          <property name="value">
           <double>0.001000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>norm_Itr:</string>
          </property>
          <property name="buddy">
           <cstring>BoxMSATalpha</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="BoxMSATNormItr">
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.000100000000000</double>
          </property>
          <property name="value">
           <double>30.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>ver_Itr:</string>
          </property>
          <property name="buddy">
           <cstring>BoxMSATalpha</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="BoxMSATVerItr">
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.000100000000000</double>
          </property>
          <property name="value">
           <double>30.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>threshold:</string>
          </property>
          <property name="buddy">
           <cstring>BoxMSATalpha</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QDoubleSpinBox" name="BoxMSATthreshold">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.000100000000000</double>
          </property>
          <property name="value">
           <double>0.900000000000000</double>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>edges visible:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QCheckBox" name="checkBox_showFeatureEdge">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>vertices visible:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QCheckBox" name="checkBox_showFeatureVertex">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonRunMSAT">
        <property name="text">
         <string>run MSAT</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButtonFeatureGraph">
        <property name="text">
         <string>feature graph</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>pushButtonRunMSAT</zorder>
   <zorder></zorder>
   <zorder>label_8</zorder>
   <zorder>checkBox_showFeatureVertex</zorder>
   <zorder>pushButtonFeatureGraph</zorder>
   <zorder>pushButtonFeatureGraph</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
